pkg/login/social/common.go

```diff
@@ -57,3 +67,30 @@ func HttpGet(client *http.Client, url string) (response HttpGetResponse, err err
 	err = nil
 	return
 }
+
+func (s *SocialBase) searchJSONForAttr(attributePath string, data []byte) string {
+	if attributePath == "" {
+		s.log.Error("No attribute path specified")
+		return ""
+	}
+	if len(data) == 0 {
+		s.log.Error("Empty user info JSON response provided")
+		return ""
+	}
+	var buf interface{}
+	if err := json.Unmarshal(data, &buf); err != nil {
+		s.log.Error("Failed to unmarshal user info JSON response", "err", err.Error())
```

Comment:
Doesn't seem right to me to swallow the error, instead of propagating it to the caller.